versioned_docs/version-8.7/components/camunda-integrations/sap/csap-cli.md

```diff
@@ -0,0 +1,5 @@
+---
+id: csap-cli
```

Comment:
Will need to add this to SAP integration overview page.

Reply:
kk, will do